config/locales/en.yml

```diff
@@ -1095,6 +1096,10 @@ en:
       usage_limit:
         value: Max. value
 
+      user:
+        first_name: 'First name'
+        last_name: 'Last name'
```

Comment:
How is this used? I see `= form.input :first_name, required: true, label: 'First name'` above directly set.

Reply:
I think at some point I mixed both approaches. The label is to be picked from `en.yaml` by formtastic, so the lines you refer are probably outdated. I will update it.